sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1504 @@
+    "gather": {
+     "logged": 1670411762609
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Save the DataFrame objects to files\n",
+    "train_data_path = \"./data/train_S4248SM144SCEN.csv\"\n",
+    "test_data_path = \"./data/test_S4248SM144SCEN.csv\"\n",
+    "df_train.to_csv(train_data_path, index=False)\n",
+    "df_test.to_csv(test_data_path, index=False)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670411762842
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Create the Azure Machine Learning MLTable"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "With Azure Machine Learning MLTables you can keep a single copy of data in your storage, easily access data during model training, share data and collaborate with other users. \n",
+    "Below, we will upload the data by creating an MLTable to be used for training.\n",
+    "\n",
+    "**NOTE:** In this PRIVATE PREVIEW we're defining the MLTable in a separate folder and .YAML file.\n",
+    "In later versions, you'll be able to do it all in Python APIs."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "def create_folder_and_ml_table(csv_file, output, delimiter=\",\", encoding=\"ascii\"):\n",
+    "    os.makedirs(output, exist_ok=True)\n",
+    "    fname = os.path.split(csv_file)[-1]\n",
+    "\n",
+    "    mltable = {\n",
+    "        \"paths\": [{\"file\": f\"./{fname}\"}],\n",
+    "        \"transformations\": [\n",
+    "            {\"read_delimited\": {\"delimiter\": delimiter, \"encoding\": encoding}}\n",
+    "        ],\n",
+    "    }\n",
+    "    with open(os.path.join(output, \"MLTable\"), \"w\") as f:\n",
+    "        f.write(yaml.dump(mltable))\n",
+    "    shutil.copy(csv_file, os.path.join(output, fname))"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670411763109
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "train_mltable_path = \"./data/training-mltable-folder\"\n",
+    "create_folder_and_ml_table(train_data_path, train_mltable_path)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670411763508
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Training MLTable defined locally, with local data to be uploaded\n",
+    "my_training_data_input = Input(type=AssetTypes.MLTABLE, path=train_mltable_path)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1670411763708
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "To create data input from TabularDataset created using V1 sdk, specify the `type` as `AssetTypes.MLTABLE`, `mode` as `InputOutputModes.DIRECT` and `path` in the following format `azureml:<tabulardataset_name>:<version>`. "
```

Comment:
Hi Nikolay, this format azureml:<tabulardataset_name>:<version> is also in the notebooks of other verticals. Would it be better if I reframe it as : "To create data input from TabularDataset created using V1 sdk, set the type to AssetTypes.MLTABLE, mode to InputOutputModes.DIRECT and path to the following format azureml:<tabulardataset_name> or azureml:<tabulardataset_name>:<version> (in case we want to use specific version). ".
Please let me know.

Reply:
Yes, we can use this wording.